docs/en_US/install/3ds.md

```diff
@@ -5,6 +5,11 @@
 	SYSTEM TRANSFERS ARE NOT CURRENTLY SUPPORTED BY OUR SERVERS. ATTEMPTING TO PERFORM A SYSTEM TRANSFER MAY PREVENT YOU FROM BEING ABLE TO GO ONLINE IN THE FUTURE. SUPPORT FOR SYSTEM TRANSFERS IS IN DEVELOPMENT.
 </div>
 
+<div class="tip red">
+	<strong>CAUTION:</strong>
+	Launching Nintendo Badge Arcade while connected to one network with badges obtained while connected to the other will cause your local save data to not match your server save data. This will cause your badges to disappear.
```

Comment:
```suggestion
	Collecting badges in Nintendo Badge Arcade while connected to one network and then launching the game on a different network will result in your badges disappearing. This occurs because the locally saved data does not match the data stored on the server.
```

Reply:
Seems good, looks to me that it still conveys the message great.